mmv1/products/compute/TargetHttpsProxy.yaml

```diff
@@ -134,7 +134,8 @@ properties:
     name: 'certificateManagerCertificates'
     description: |
       URLs to certificate manager certificate resources that are used to authenticate connections between users and the load balancer.
-      Currently, you may specify up to 15 certificates. Certificate manager certificates do not apply when the load balancing scheme is set to INTERNAL_SELF_MANAGED.
+      Currently, you may specify up to 15 certificates. Certificate manager certificates only apply when the load balancing scheme is set to INTERNAL_MANAGED.
```

Comment:
"Currently, you may specify up to 15 certificates." this is not true for CertificateManagerCertificates.

```
URLs to Certificate manager certificate resources that are used to authenticate connections between users and the load balancer.
      Certificate manager certificates only apply when the load balancing scheme is set to INTERNAL_MANAGED.
      SslCertificates and CertificateManagerCertificates can not be defined together.
      ```

Reply:
Done! Just to confirm, the 15 certificate limit is still true for sslCertificates and not for certificate manager certificate, right?

Reply:
Yes, exactly.